csharp/src/Google.Protobuf.Benchmarks/Google.Protobuf.Benchmarks.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
```

Comment:
how important is this? The existing test infrastructure currently has dotnet SDK 2.1

Reply:
The package has a new netstandard2.1 target so we can use types like SequenceReader and ArrayBufferWriter without copying them.

netcoreapp3.0 is needed to test netstandard2.1.